### PR TITLE
New UT + Code improvment

### PR DESCRIPTION
**Pull request by oualib**
- More UT
 - assertion instead of conditional
 - correction of some bugs
 - removing the deprecated directories

### CODE REVIEW

**Comment by oualib** [comment]
@afard can you please check and approve this PR, so I can know what I should cover next?

File: verticapy/tests/base.py [hunk 1]
```diff
@@ -20,6 +20,7 @@
 import inspect
 import getpass
 import vertica_python
+import verticapy
```
Comment (afard) [inline verticapy/tests/base.py:23]: I think this import is redundant. Let's avoid redundant imports.

File: verticapy/tests/vModel/test_naive_bayes.py [hunk 2]
```diff
@@ -43,6 +43,16 @@ def model(base, iris_vd):
 
 
 class TestNB:
+    def test_NB_subclasses(self):
+        result = BernoulliNB("model_test")
+        result.parameters["nbtype"] == "bernoulli"
```
Comment (afard) [inline verticapy/tests/vModel/test_naive_bayes.py:48]: I think you forgot to assert.

File: verticapy/tests/vModel/test_naive_bayes.py [hunk 3]
```diff
@@ -43,6 +43,16 @@ def model(base, iris_vd):
...
+        result = CategoricalNB("model_test")
+        result.parameters["nbtype"] == "categorical"
```
Comment (afard) [inline verticapy/tests/vModel/test_naive_bayes.py:50]: assert?

File: verticapy/tests/vModel/test_naive_bayes.py [hunk 4]
```diff
@@ -43,6 +43,16 @@ def model(base, iris_vd):
...
+        result = GaussianNB("model_test")
+        result.parameters["nbtype"] == "gaussian"
```
Comment (afard) [inline verticapy/tests/vModel/test_naive_bayes.py:52]: assert?

File: verticapy/tests/vModel/test_naive_bayes.py [hunk 5]
```diff
@@ -43,6 +43,16 @@ def model(base, iris_vd):
...
+        result = MultinomialNB("model_test")
+        result.parameters["nbtype"] == "multinomial"
```
Comment (afard) [inline verticapy/tests/vModel/test_naive_bayes.py:54]: assert?

File: verticapy/tests/vModel/test_normalizer.py [hunk 6]
```diff
@@ -38,6 +38,14 @@ def model(base, winequality_vd):
 
 
 class TestNormalizer:
+    def test_Normalizer_subclasses(self):
+        result = StandardScaler("model_test")
+        result.parameters["method"] == "zscore"
```
Comment (afard) [inline verticapy/tests/vModel/test_normalizer.py:43]: assert?

File: verticapy/tests/vModel/test_normalizer.py [hunk 7]
```diff
@@ -38,6 +38,14 @@ def model(base, winequality_vd):
...
+        result = RobustScaler("model_test")
+        result.parameters["method"] == "robust_zscore"
```
Comment (afard) [inline verticapy/tests/vModel/test_normalizer.py:45]: assert?

File: verticapy/tests/vModel/test_normalizer.py [hunk 8]
```diff
@@ -38,6 +38,14 @@ def model(base, winequality_vd):
...
+        result = MinMaxScaler("model_test")
+        result.parameters["method"] == "minmax"
```
Comment (afard) [inline verticapy/tests/vModel/test_normalizer.py:47]: assert?

File: verticapy/tests/sql/test_sql.py [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+        d = read_dsn("vp_test_config", os.path.dirname(verticapy.__file__) + "/tests/verticaPy_test_tmp.conf",)
+        new_auto_connection(d, "VerticaDSN_test")
+        change_auto_connection("VerticaDSN_test")
+        result = sql("-limit 30", "SELECT * FROM titanic;")
+        assert len(result["age"]) == 30
+        result = sql("-vdf True", "SELECT * FROM titanic;")
+        assert result.shape() == (1234, 14)
+        result = sql("-vdf True", "DROP MODEL IF EXISTS model_test; SELECT LINEAR_REG('model_test', 'public.titanic', 'survived', 'age, fare'); SELECT PREDICT_LINEAR_REG(3.0, 4.0 USING PARAMETERS model_name='model_test', match_by_pos=True) AS predict;")
+        assert result["predict"][0] == pytest.approx(0.395335892040411)
+        result = sql("-limit 30", "DROP MODEL IF EXISTS model_test;")
```
Comment (afard) [inline verticapy/tests/sql/test_sql.py:41]: Have you forgotten to add assert for the last result?

**Comment by afard** [comment]
@oualib, I reviewed the PR, and left some comments. I will approve it after you address them.

File: verticapy/tests/base.py (hunk 1 above)
Comment (oualib) [reply verticapy/tests/base.py:23]: verticapy != vertica_python

File: verticapy/tests/vModel/test_naive_bayes.py (hunk 3 above)
Comment (oualib) [reply verticapy/tests/vModel/test_naive_bayes.py]: Done;

File: verticapy/tests/vModel/test_naive_bayes.py (hunk 4 above)
Comment (oualib) [reply verticapy/tests/vModel/test_naive_bayes.py]: Done;

File: verticapy/tests/vModel/test_naive_bayes.py (hunk 5 above)
Comment (oualib) [reply verticapy/tests/vModel/test_naive_bayes.py]: Done;

File: verticapy/tests/vModel/test_normalizer.py (hunk 6 above)
Comment (oualib) [reply verticapy/tests/vModel/test_normalizer.py]: Done;

File: verticapy/tests/vModel/test_normalizer.py (hunk 7 above)
Comment (oualib) [reply verticapy/tests/vModel/test_normalizer.py]: Done;

File: verticapy/tests/vModel/test_normalizer.py (hunk 8 above)
Comment (oualib) [reply verticapy/tests/vModel/test_normalizer.py]: Done;

File: verticapy/tests/sql/test_sql.py (hunk 9 above)
Comment (oualib) [reply verticapy/tests/sql/test_sql.py]: Done;

File: verticapy/tests/vModel/test_naive_bayes.py (hunk 2 above)
Comment (oualib) [reply verticapy/tests/vModel/test_naive_bayes.py]: Done;

File: verticapy/tests/base.py (hunk 1 above)
Comment (afard) [reply verticapy/tests/base.py:23]: I know :), but `importing verticapy` looks redundant while the content of the file is not changed. Have you tried to run the tests without this import?

Comment (oualib) [reply verticapy/tests/base.py]: Done;

**Pull request merged**